examples/quickstarts/helidon-quickstart-se/src/main/docker/Dockerfile.native

```diff
@@ -20,21 +20,21 @@ WORKDIR /helidon
 ADD *.jar target/
 ADD ${libs.classpath.prefix}/*.jar target/
 
-RUN echo "$GRAALVM_HOME/bin/native-image -H:Name=helidon-native -H:IncludeResources='application.yaml|logging.properties' -H:+ReportExceptionStackTraces -classpath target/* ${main.class}"
-RUN $GRAALVM_HOME/bin/native-image \
+RUN echo "${GRAALVM_HOME}/bin/native-image -H:Name=${project.artifactId}-native -H:IncludeResources='application.yaml|logging.properties' -H:+ReportExceptionStackTraces -classpath target/* ${mainClass}"
```

Comment:
This restricts the resources to be always application.yaml and logging.properties. 
Also for a general application, it may make sense to add configuration file such as application-graal.yaml that would be configured as the first configuration file, so we could set things such as port to be explicitly the same as the one in Dockerfile

Reply:
Same comment as above this will be removed once I show proper multi-stage docker build. There won't be such duplicated configuration.